src/js/apps/patients/worklist/filters_app.js

```diff
@@ -69,7 +84,7 @@ export default App.extend({
     const groups = this.groups.clone();
 
     groups.unshift({
-      id: 'all-groups',
+      id: null,
       name: 'All Groups',
```

Comment:
Should this be i18n-ified?

Reply:
yeah you're right it should 100% be i18n'd